viewing_party/main.py

```diff
@@ -0,0 +1,139 @@
+def create_movie(title,genre,rating):
+    new_movie = {
+        "title": "",
+        "genre": "",
+        "rating": 0
+    }
+    if title and genre and rating:
+        new_movie["title"] = title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+        return new_movie
+    else:
+        return None
+
+
+def add_to_watched(user_data,movie):
+    for key, value in user_data.items():
+        user_data[key].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    for key, value in user_data.items():
+        user_data[key].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for movies in user_data["watchlist"]:
+        if title == movies["title"]:
+            user_data["watched"].append(movies)
+            user_data["watchlist"].pop(0)
```

Comment:
Using the index to remove an item works now, however, what happens if the movie is in a different index?